packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
We did not have named imports in CFD package until now. Better stick to it and use `React.useEffect`.
And beside that, why do you need to import it when you do not use it.